### ISSUE TITLE
Does glmnetplus contain any screening rules?

### ISSUE THREAD

**Issue by prateeksasan1** [junyangq/glmnetPlus#7]
Hi,

I wanted to know whether any safe/strong screening rules have already been implemented in glmnetplus for family = "mgaussian"? I am trying to run an elastic net penalty for "mgaussian" (alpha = 0.99) and want to get a sense of what coding i would have to do on top of glmnetplus

Thanks
Prateek Sasan

**Comment by junyangq** [comment]
Yes, it includes strong rules screening. For mgaussian family, almost everything is the same as glmnet except that it opens up the option for one to provide his or her own initialization.
